Samples/DirectMLNpuInference/main.cpp

```diff
@@ -88,8 +88,14 @@ void InitializeDirectML(ID3D12Device1** d3dDeviceOut, ID3D12CommandQueue** comma
     if (adapter)
     {
         std::string adapterName;
-        TryGetProperty(adapter, DXCoreAdapterProperty::DriverDescription, adapterName);
-        printf("Successfully found adapter %s\n", adapterName.c_str());
+        if (TryGetProperty(adapter.Get(), DXCoreAdapterProperty::DriverDescription, adapterName))
+        {
+            printf("Successfully found adapter %s\n", adapterName.c_str());
+        }
+        else
+        {
+            printf("Failed to get adapter name.\n");
```

Comment:
Just like you do on the line above, it's good to print the name of the adapter so the user can see exactly what they asked for that failed. Including little quotes is also nice, in case there is ever some weirdness related to leading/trailing whitespace:

```suggestion
            printf("Failed to get adapter name '%s'\n", adapterName.c_str());
```

Reply:
Ah! If I understand your comment correctly, in this particular case what we are failing to get is the adapterName, so I expect c_str to be undefined or an empty string here. I changed to "Failed to get adapter description" to be less confusion.

Reply:
Oops, I conflated this with some related adapter lookup code (in DML) where we were finding an adapter by name, which I thought by looking at the error message, but this is clearly *getting* the name of a given adapter. Yes, the new error message is clearer.

Reply:
> I changed to "Failed to get adapter description" to be less confusion.

Hmm, this comment is resolved, but I don't actually see that change?